src/reanimated2/animation/sequence.ts

```diff
@@ -11,8 +11,18 @@ export function withSequence(
   ..._animations: NextAnimation<AnimationObject>[]
 ): Animation<SequenceAnimation> {
   'worklet';
+
+  const animationFunctionCall = {
+    functionName: 'withSequence',
+    // eslint-disable-next-line prefer-rest-params
+    functionArguments: [...arguments],
+    // eslint-disable-next-line prefer-rest-params
+    animatedArgumentsIndices: [...arguments].map((_, i) => i),
```

Comment:
Not sure if we need to copy `arguments` in this case since we call `map` afterwards.

Reply:
As I wrote above `arguments` is not an array and it has no `map` function.